### PR TITLE
fix/heatmap-colors

### LINKED ISSUE #36650: [.dev] heatmap probably needs a color update
Screen Shot 2019-08-20 at 12 37 39 PM

### PR DESCRIPTION
**Pull request by moT01**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #36650

Previews:
Screen Shot 2019-08-21 at 11 06 54 AM
Screen Shot 2019-08-21 at 11 06 42 AM

### CODE REVIEW

**Review by thecodingaviator — COMMENTED** [review]
This looks amazing 🔥

**Comment by jonathan-grah** [comment]
@moT01 😍

File: client/src/components/profile/components/HeatMap.js [hunk 1]
```diff
@@ -57,12 +57,18 @@ function HeatMap({ calendar, streak }) {
...
+              return 'color-scale-1';
             }
-            return `colour-scale-${value.count}`;
+            if (value.count < 8) {
+              return 'color-scale-2';
+            }
+            if (value.count >= 8) {
+              return 'color-scale-a-lot';
+            }
+            return 'color-scale-empty';
```
Comment (ojeytonwilliams) [inline client/src/components/profile/components/HeatMap.js]:
```suggestion
            return 'color-empty';
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Nice one, @moT01, this looks cool!

I found one typo, see below.

Also, one thought I had was, the chain of `if`s might look a bit cleaner if you got rid of the `{}` since they're each one liners

```js
if(cond) return 'result'
```
That sort of thing.  Not a huge deal, I'd be happy merging if the typo's fixed, but I figured I'd mention it.

File: client/src/components/profile/components/HeatMap.js (hunk 1 above)
Comment (moT01) [reply client/src/components/profile/components/HeatMap.js]: nice catch

**Review by ojeytonwilliams — APPROVED** [review]

**Comment by ojeytonwilliams** [comment]
Thanks, @moT01, it looks nice and tidy now.

**Comment by moT01** [comment]
Is there something specific you want updated here @raisedadead?

**Comment by ahmadabdolsaheb** [comment]
@moT01, this pr is in the queue. I will test it locally in a day or two and let you know.

**Review by raisedadead — APPROVED** [review]

**Pull request merged**